packages/node-config-provider/src/index.ts

```diff
@@ -1 +1,4 @@
 export * from "./configLoader";
+export { EnvOptions, GetterFromEnv } from "./fromEnv";
+export { GetterFromConfig, SharedConfigInit } from "./fromSharedConfigFiles";
+export { FromStaticConfig } from "./fromStatic";
```

Comment:
There should be a form of "internal package, do not use" in the top level readme.